### DISCUSSION TITLE
Access Issue

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [andrewdwallo/filament-companies#60] [Q&A] [answered]
I have Project with Shield Package 

![image](https://user-images.githubusercontent.com/47450090/229576001-314ec6dd-4a0a-4029-9234-8bed08d2532e.png)

When I disable the permissions for the pages nothing changes and I can access them

**Comment by andrewdwallo** [comment, marked as answer]
The roles and permissions of this package are not based on the shield package... I can't help you here.
